dropwizard-views/src/main/java/io/dropwizard/views/ViewMessageBodyWriter.java

```diff
@@ -82,12 +76,9 @@ public void writeTo(View t,
                 }
             }
             throw new ViewRenderException("Unable to find a renderer for " + t.getTemplateName());
-        } catch (Exception e) {
+        } catch (ViewRenderException e) {
             LOGGER.error("Template Error", e);
```

Comment:
My preference would be to format the log message in my own ExceptionMapper, so could this go into the ExceptionMapper as well?